### PR TITLE
[CARBONDATA-1342] Fixed bugs for spark conf property and debugging in windows

### PR DESCRIPTION
**Pull request by manishgupta88**
Fixes include:
1. In spark 2, spark conf once set in spark context cannot be modified with the same context again. Therefore removed setting property in spark conf and directly getting the property from carbon properties.
2. Fixed bug of running CarbonSessionExample in windows by removing File.Separator
3. Removed call of namenode after completion of each data load for renaming bad folders

### CODE REVIEW

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6, Please check CI http://144.76.159.231:8080/job/ApacheCarbonPRBuilder/676/

**Comment by CarbonDataQA** [comment]
Can one of the admins verify this patch?

File: core/src/main/java/org/apache/carbondata/core/metadata/AbsoluteTableIdentifier.java [hunk 1]
```diff
@@ -99,8 +98,9 @@ public static AbsoluteTableIdentifier fromTablePath(String tablePath) {
   }
 
   public String getTablePath() {
-    return getStorePath() + File.separator + getCarbonTableIdentifier().getDatabaseName() +
-        File.separator + getCarbonTableIdentifier().getTableName();
+    return getStorePath() + CarbonCommonConstants.FILE_SEPARATOR + getCarbonTableIdentifier()
```
Comment (xuchuanyin) [inline core/src/main/java/org/apache/carbondata/core/metadata/AbsoluteTableIdentifier.java]: why need to change this? In another modification in this PR, you still use `File.separator` in `renameBadRecord`.

Comment (manishgupta88) [inline core/src/main/java/org/apache/carbondata/core/metadata/AbsoluteTableIdentifier.java]: I have modified in renameBadRecord method also to use CarbonCommonConstants.FILE_SEPARATOR in place of File.separator

**Comment by ravipesala** [comment]
SDV Build Failed with Spark 2.1, Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/218/

**Comment by ravipesala** [comment]
SDV Build Success with Spark 2.1, Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/221/

**Comment by ravipesala** [comment]
LGTM

**Comment by manishgupta88** [comment]
Already merged

**Pull request closed without merging**